### ISSUE TITLE
No `should_close` method on `glutin::window::Window`

### ISSUE THREAD

**Issue by mitchmindtree** [PistonDevelopers/glutin_window#27]
It must have been moved or replaced, will look into it now.

Here's the error:

```
src/lib.rs:144:21: 144:35 error: type `glutin::window::Window` does not implement any method in scope named `should_close`
src/lib.rs:144         self.window.should_close() || self.should_close                                                    
```

**Comment by mitchmindtree** [comment]
Seems it was replaced with `is_closed`, will fix now

**Comment by mitchmindtree** [comment]
closed by #28
